README.md

```diff
@@ -1,22 +1,70 @@
-Mailscript CLI
-==============
+<p align="center">
+  <img src="docs/media/Mailscript_Black.png">
+</p>
+<p align="center">
+  <H1>Low-code automation for inbound emails.</H1>
+ <H4> Get a programmable email address. Automate what happens when you receive emails. It's like Zapier for devs who hate emails.</H4>
+</p>
 
-Configure and use mailscript from the command-line
+<div align="center">
+
+<hr/>
 
 [![Version](https://img.shields.io/npm/v/mailscript.svg)](https://npmjs.org/package/mailscript)
 [![Downloads/week](https://img.shields.io/npm/dw/mailscript.svg)](https://npmjs.org/package/mailscript)
 [![License](https://img.shields.io/npm/l/mailscript.svg)](https://github.com/mailscript/mailscript/blob/master/package.json)
-
+![Main](https://github.com/mailscript/cli/workflows/Main/badge.svg)
+[![GitHub Issues](https://img.shields.io/github/issues/mailscript/cli.svg)](https://github.com/mailscript/cli/issues)
+![Contributions welcome](https://img.shields.io/badge/contributions-welcome-orange.svg)
+[![Discord](https://img.shields.io/discord/475789330380488707?color=blueviolet&label=discord&style=flat-square)](https://discord.gg/US24HAVYq2)
+<a href="https://twitter.com/intent/follow?screen_name=getmailscript">
+  <img src="https://img.shields.io/twitter/follow/getmailscript.svg?label=Follow%20@getmailscript" alt="Follow @getmailscript" />
+</a>
+
+</div>
+
+<p align="center">
+  <a href="https://vimeo.com/489472356">
+    <img src="docs/media/ms_1.png">
+  </a>
+<p>
+
+# Table of Contents 📚
 <!-- toc -->
+* [Features](#features)
```

Comment:
These links are not working, it must have something to do with the section titles having space and an emoji; when I click directly on the title anchor elements the links show an extra `-` in the url bar which the menu anchors do not have. Minor details but let's fix it.

Reply:
Fixed: Need `-(emoji)`